src/server/rpc/procedures/air-quality/air-quality.js

```diff
@@ -9,94 +9,106 @@
 // for download online.
 'use strict';
 
-const logger = require('../utils/logger')('air-quality');
-const API_KEY = process.env.AIR_NOW_KEY;
-const path = require('path');
-const fs = require('fs');
-const geolib = require('geolib');
-const request = require('request');
-
-var baseUrl = 'http://www.airnowapi.org/aq/forecast/zipCode/?format=application/' + 
-        'json&API_KEY=' + API_KEY,
-    reportingLocations = (function() {  // Parse csv
-        var locationPath = path.join(__dirname, 'air-reporting-locations.csv'),
-            text = fs.readFileSync(locationPath, 'utf8'),
-            rawLocations = text.split('\n');
-
-        rawLocations.pop();  // Remove trailing \n
-        rawLocations.shift();  // Remove header
-        return rawLocations
-            .map(function(line) {
-                var data = line.split('|');
-                return {
-                    city: data[0],
-                    state: data[1],
-                    zipcode: data[2],
-                    latitude: +data[3],
-                    longitude: +data[4]
-                };
-            });
-    })();
-
-
-var getClosestReportingLocation = function(lat, lng) {
-    var nearest = geolib.findNearest({latitude: lat, longitude: lng}, reportingLocations),
+const ApiConsumer = require('../utils/api-consumer');
+
+var debug = require('debug'),
+    error = debug('netsblox:rpc:air-quality:error'),
```

Comment:
if you merge master in, these should be replaced with a single `logger` object which was updated on master to ensure uniformity in RPC logging (https://github.com/NetsBlox/NetsBlox/issues/2071).

Reply:
That's done now